artifactory.py

```diff
@@ -1079,7 +1086,7 @@ def open(self, pathobj):
         response = self.get_response(pathobj)
         return response.raw
 
-    def get_response(self, pathobj):
+    def get_response(self, pathobj, skip_quote_url=False):
```

Comment:
Let's do an opposite, `quote=True` 
It's better to use positive flags where it's possible, easy to read

```python
# two negatives 
if not skip_quote_url:
    url = quote_url(url)

# VS
if quote:
    url = quote_url(url)
```

Reply:
ok, thanks for your advice.